### PR TITLE
Added a README.md & test.properties file to the wycheproof docs

### PR DESCRIPTION
**Pull request by Annysah**

### CODE REVIEW

**Comment by Annysah** [comment]
@karianna @smlambert Kindly review my PR.

File: external/wycheproof/test.properties [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+        tag_version="master"
+        bazel_version="1.2.1"
+        debian_packages="git wget unzip zip g++"
+        debianslim_packages="${debian_packages}"
+        ubuntu_packages="${debian_packages}"
+        alpine_packages="bash git wget unzip zip g++ bash"
+        centos_packages="git wget unzip zip gcc-c++"
+        clefos_packages="${centos_packages}"
+        ubi_packages="git wget unzip zip gcc-c++"
+        ubi_minimal_packages="${ubi_packages}"
```
Comment (smlambert) [inline external/wycheproof/test.properties:481]: We only need this section for the test.properties.  Take a look at https://github.com/AdoptOpenJDK/openjdk-tests/pull/2452/files for a similar example.

File: external/wycheproof/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Running External tests locally
```
Comment (smlambert) [inline external/wycheproof/README.md:1]: Please add a Heading and paragraph (similar to https://github.com/AdoptOpenJDK/openjdk-tests/pull/2452/files):

External Wycheproof Tests

Wycheproof tests are a part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Wycheproof test material is pulled from the [wycheproof](https://github.com/google/wycheproof.git) repository.

File: external/wycheproof/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+0. Ensure your test machine is set up with [test prereqs](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/Prerequisites.md).  For external tests, you do need Docker installed.
+
+1. Download/unpack the SDK you want to your test machine
+2. `export TEST_JDK_HOME=</pathToWhereYouInstalledSDK>` 
+3. `git clone https://github.com/AdoptOpenJDK/openjdk-tests.git` 
+4. `cd openjdk-tests`
+5. `./get.sh`
+6. `cd TKG`
+7. export required environment variables, BUILD_LIST and EXTRA_DOCKER_ARGS (example: `export BUILD_LIST=external/jacoco` and `export EXTRA_DOCKER_ARGS="-v $TEST_JDK_HOME:/opt/java/openjdk"`
```
Comment (smlambert) [inline external/wycheproof/README.md:12]: BUILD_LIST=external/wycheproof

File: external/wycheproof/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+8. `make wycheproof`        (This fetches test material and compiles it, based on build.xml files in the test directories)
+9. `make wycheproof_test`   (When you defined BUILD_LIST to point to a directory in [openjdk-tests/external](https://github.com/AdoptOpenJDK/openjdk-tests/tree/master/external), then this is a testCaseName from the playlist.xml file within the directory you chose)
```
Comment (smlambert) [inline external/wycheproof/README.md:14]: Please add an extra paragraph at the end of the README: 

When [running these from the command-line](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/userGuide.md#local-testing-via-make-targets-on-the-commandline), these tests are grouped under a make target called 'external', so 'make external' would run the entire set of tests found in the openjdk-tests/external directory. This is unadvisable! Limit what you compile and run, BUILD_LIST=external/`<someSubDirectory>`, and TARGET=`<testCaseNameFromSubdirPlaylist>`

These tests run regularly and results can be found in [TRSS Third Party Application view](https://trss.adoptopenjdk.net/ThirdPartyAppView).

**Review by smlambert — COMMENTED** [review]
hi @Annysah - some updates requested.

File: external/wycheproof/test.properties (hunk 1 above)
Comment (Annysah) [reply external/wycheproof/test.properties:481]: Thanks @smlambert for the review, working on the updates requested.

**Comment by Annysah** [comment]
@smlambert  The requested updates has been made, kindly review.

File: external/wycheproof/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+6. `cd TKG`
+
+7. export required environment variables, BUILD_LIST and EXTRA_DOCKER_ARGS (example: `export BUILD_LIST=external/wycheproof` and `export EXTRA_DOCKER_ARGS="-v $TEST_JDK_HOME:/opt/java/openjdk"`
+
+8. `make wycheproof`        (This fetches test material and compiles it, based on build.xml files in the test directories)
+
+9. `make wycheproof_test`   (When you defined BUILD_LIST to point to a directory in [openjdk-tests/external](https://github.com/AdoptOpenJDK/openjdk-tests/tree/master/external), then this is a testCaseName from the playlist.xml file within the directory you chose)
+
+When [running these from the command-line](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/userGuide.md#local-testing-via-make-targets-on-the-commandline), these tests are grouped under a make target called 'external', so 'make external' would run the entire set of tests found in the openjdk-tests/external directory. This is unadvisable! Limit what you compile and run, BUILD_LIST=external/<someSubDirectory>, and TARGET=<testCaseNameFromSubdirPlaylist>
```
Comment (sophia-guo) [inline external/wycheproof/README.md:29]: The last sentence you will need to add `` before < and after >, to make the context visible. That is BUILD_LIST=external/`<someSubDirectory>`, and TARGET=`<testCaseNameFromSubdirPlaylist>`.

And I would suggest to update as BUILD_LIST=external/wycheproof, and TARGET=`<testCaseNameFromPlaylistUnderSubdirwycheproof>`

Comment (Annysah) [reply external/wycheproof/README.md:29]: Thanks @sophia-guo for pointing this out, making the changes ASAP.

Comment (Annysah) [reply external/wycheproof/README.md:29]: Should the wycheproof in this sentence be in angle brackets as well like you did for "<testCaseNameFromPlaylistUnderDirwycheproof>"  BUILD_LIST=external/wycheproof, and TARGET=<testCaseNameFromPlaylistUnderDirwycheproof>

Comment (sophia-guo) [reply external/wycheproof/README.md:29]: As it is a specific subdirectory so i think no need to put the angle brackets.

Comment (Annysah) [reply external/wycheproof/README.md:29]: Alright, thanks.

Comment (Annysah) [reply external/wycheproof/README.md]: Hi @sophia-guo, The required changes has been made, kindly review.

**Review by sophia-guo — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]
thanks @Annysah !

**Comment by Annysah** [comment]
Yaay!
Thanks @smlambert & @sophia-guo

**Pull request merged**